log-analyzer/features/org.wso2.carbon.la.apim.feature/src/main/resources/gadgets/LogAPIMArtifactDeleted/js/moment.js

```diff
@@ -0,0 +1,2808 @@
+//! moment.js
+//! version : 2.8.1
```

Comment:
Hi Sinthuja,

We have handle this by adding those common js files in common gadget folder and other gadget refer all the common js and css files from there , relevant commit for these fixes are in https://github.com/wso2/shared-analytics/pull/24/commits/478aa19f4c3da194febf2ef048c5092d7ee2057c#diff-54c81dad3ef934b6e7c173185a3aad21R55

Thanks,
SajithD